api.py

```diff
@@ -0,0 +1,82 @@
+from flask import Flask, request, jsonify,render_template,send_from_directory
+from dotenv import load_dotenv
+from App import App
+import asyncio
+import os
+
+load_dotenv()
+
+app = Flask(__name__)
+bot = App()
+
+
+@app.route('/')
+def home():
+    return render_template('home.html')
+    
+#CREATE
+@app.route('/upload',methods=['POST'])
+def handle_Post():
+    if request.method == 'POST':
+        if 'file' not in request.files:
+            return jsonify({"message": "No file part in the request"}), 400
+        
+        user_file = request.files['file']
+
+        if user_file.filename == '':
+            return jsonify({"message": "No selected file"}), 400
+        else:
+            user_file.save(os.path.join(bot.file_manager.temp_dir, user_file.filename))
+            asyncio.run(bot.upload_file(user_file.filename))
+            bot.file_manager.clean_directory()
+            return jsonify({"message": "File successfully uploaded"}), 200    
+
+#  READ 
+@app.route('/uploads/', methods=['GET'])
+@app.route('/uploads/<int:Uid>',methods=['GET'])
```

Comment:
Merge with delete

Reply:
and download. make it look beautiful. make all stuff in snake_case look normally